@navikt/core/tokens/darkside/tokens/breakpoint.ts

```diff
@@ -3,17 +3,72 @@ import { type StyleDictionaryToken } from "../tokens.util";
 
 export const breakpointTokenConfig = {
   breakpoint: {
-    xs: { value: "0", type: "global-breakpoint" },
-    sm: { value: "480px", type: "global-breakpoint" },
-    "sm-down": { value: "479px", type: "global-breakpoint" },
-    md: { value: "768px", type: "global-breakpoint" },
-    "md-down": { value: "767px", type: "global-breakpoint" },
-    lg: { value: "1024px", type: "global-breakpoint" },
-    "lg-down": { value: "1023px", type: "global-breakpoint" },
-    xl: { value: "1280px", type: "global-breakpoint" },
-    "xl-down": { value: "1279px", type: "global-breakpoint" },
-    "2xl": { value: "1440px", type: "global-breakpoint" },
-    "2xl-down": { value: "1439px", type: "global-breakpoint" },
+    xs: {
+      value: "0",
+      type: "global-breakpoint",
+      group: "mobile first",
+      comment: "TODO: Sjur fyller ut",
```

Comment:
The "TODO: Sjur fyller ut" may be removed, but we agreed to keep them as placeholders for now.
They are [filtered out in style-dictionary.formats.js](https://github.com/navikt/aksel/pull/3747/files#:~:text=comment%3A%20token.comment%3F.,%3A%20token.comment%2C).